serving/src/main/java/ai/djl/serving/http/ClusterRequestHandler.java

```diff
@@ -64,14 +64,26 @@ protected void handleRequest(
             QueryStringDecoder decoder,
             String[] segments)
             throws ModelException {
+        Path home = Paths.get(System.getProperty("user.home")).resolve(".ssh");
         switch (segments[2]) {
-            case "sshkey":
-                Path home = Paths.get(System.getProperty("user.home")).resolve(".ssh");
-                Path file = home.resolve("id_rsa.pub");
-                if (Files.notExists(file)) {
+            case "sshpublickey":
+                Path public_file = home.resolve("id_rsa.pub");
+                if (Files.notExists(public_file)) {
                     sshkeygen(home.resolve("id_rsa").toString());
                 }
-                NettyUtils.sendFile(ctx, file, false);
+                NettyUtils.sendFile(ctx, public_file, false);
+                return;
+            case "sshprivatekey":
```

Comment:
We should never send private key. If bidirection ssh is required, we should run keygen on both machine.

Reply:
Removed it all, as bi-directional communication does not seem to be needed currently.